moonshot/__main__.py

```diff
@@ -232,7 +269,6 @@ def main():
         # Handle CLI mode here, possibly also with additional arguments
         pass
     else:
-        parser.print_help()
         sys.exit(1)
```

Comment:
can we at the same time check all the sys.exit if they should be returning 1 as an error occurred or 0 as program terminated successfully